paimon-flink/paimon-flink-common/src/main/java/org/apache/paimon/flink/action/cdc/mysql/MySqlDebeziumJsonEventParser.java

```diff
@@ -200,8 +203,10 @@ public List<CdcRecord> getRecords() {
     }
 
     private Map<String, String> extractRow(JsonNode recordRow) {
-        Map<String, String> recordMap =
-                objectMapper.convertValue(recordRow, new TypeReference<Map<String, String>>() {});
+
+        // the geometry,point type can not be converted to string, so we convert it to Object first.
+        Map<String, Object> recordMap =
```

Comment:
Can we just rename this to `jsonMap`?
And create a new map `Map<String, String> resultMap`.

Reply:
I update all comments .